pyqtgraph/graphicsItems/PColorMeshItem.py

```diff
@@ -0,0 +1,253 @@
+from __future__ import division
+
+from ..Qt import QtGui, QtCore
+import numpy as np
+from .. import functions as fn
+from .. import debug as debug
+from .GraphicsObject import GraphicsObject
+from ..Point import Point
+from .. import getConfigOption
+from .GradientEditorItem import Gradients # List of colormaps
+from ..colormap import ColorMap
+
+try:
+    from collections.abc import Callable
+except ImportError:
+    # fallback for python < 3.3
+    from collections import Callable
+
+__all__ = ['PColorMeshItem']
+
+
+class PColorMeshItem(GraphicsObject):
+    """
+    **Bases:** :class:`GraphicsObject <pyqtgraph.GraphicsObject>`
+    """
+
+    sigImageChanged = QtCore.Signal()
+    sigRemoveRequested = QtCore.Signal(object)  # self; emitted when 'remove' is selected from context menu
+
+
+    def __init__(self, *args, **kwargs):
+        """
+        Create a pseudocolor plot with convex polygons.
+
+        Call signature:
+
+        PColorMeshItem([x, y,] z, **kwargs)
+
+        x and y can be used to specify the corners of the quadrilaterals.
+        z must be used to specified to color of the quadrilaterals.
+
+        Parameters
```

Comment:
Unfortunately, we haven't yet moved to NumPy style doc strings, so these will fail to build with the current sphinx config. It might be possible to just enable the `sphinx.ext.napoleon` extension and both styles would be accepted. I can look into that if you want or you can go ahead and try it.

Reply:
I added the napoleon extension in the conf.py file:
```python
extensions = ['sphinx.ext.autodoc', 'sphinx.ext.viewcode', 'sphinx.ext.napoleon']
```
Is that enough?
I run `sphinx-apidoc -f -o docs/source` but I do not now what to expect.

Reply:
Like I mentioned above, we're currently manually creating the API reference.

The good news is adding `sphinx.ext.napoleon` does just give us the ability to use numpy style docstrings without affecting how other docstrings are parsed (the others are "pure RST" so I guess that makes sense).

I'm :+1: for going ahead and committing that change to `conf.py` and then we don't have to overhaul the `PColorMeshItem` docstrings.

Reply:
ok done